docs/release.sh

```diff
@@ -78,6 +78,11 @@ upload_current_documentation() {
 	src=site/
 	dst=s3://$BUCKET$1
 
+	cache="max-age=3600"
+	if [ "$NOCACHE" ]; then
+		cache="no-cache"
```

Comment:
(and if this were in `project/` or `contrib/`, these would definitely need fixing :wink:)